guides/micronaut-openapi-generator-server/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "asciidoctor": "micronaut-openapi-generator-server.adoc",
+  "slug": "micronaut-openapi-generator-server",
+  "title": "Use OpenAPI Definition to Generate a Micronaut Server",
+  "intro": "Learn how to write an OpenAPI definition, use it to generate a server template for Micronaut framework, and get it all to work",
+  "authors": ["Andriy Dmytruk"],
+  "tags": ["openapi", "server"],
+  "category": "Getting Started",
+  "languages": ["java"],
+  "buildTools": ["maven", "gradle"],
+  "publicationDate": "2022-02-01",
```

Comment:
be sure to update the date so it's close to the date when it's published on the live site

Reply:
Also I am not sure what category this should be put in. "Getting started" might not be the right one.

Reply:
Add a new category to the enum named "API"